src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AgencyInternalLocationRepositoryExt.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository
```

Comment:
added kotlin extensions where we convert a Optional into a nullable similar to generic extension for `findById`